.kokoro/presubmit/graalvm-native-17.cfg

```diff
@@ -30,4 +30,9 @@ env_vars: {
 env_vars: {
   key: "SECRET_MANAGER_KEYS"
   value: "java-it-service-account"
+}
+
```

Comment:
Let's also merge this PR with `main` so that it is tested with GraalVM 22.3.0. `gcr.io/cloud-devrel-kokoro-resources/graalvm17` was recently upgraded to `gcr.io/cloud-devrel-kokoro-resources/graalvm17:22.3.0`

Reply:
will do - I will have a followup PR updating this branch, so stay tuned!